### ISSUE TITLE
buildah refuses to run as root on Fedora 32 Silverblue

### ISSUE THREAD

**Issue by fansari** [containers/buildah#2298]
**Description**

Any run of the buildah command gives this output. This only happens as root, not when I run buildah as normal user.

Error loading container config network config: invalid cni_plugin_dirs entry: mkdir /usr/lib/cni: read-only file system

This is clear: on Fedora Silverblue /usr is a read-only moutpoint.

buildah-1.14.0-0.31.dev.git82ff48a.fc32.x86_64

**Steps to reproduce the issue:**
1. buildah info
2. buildah images
3. or any other buildah command run as root user

**Describe the results you received:**

Error loading container config network config: invalid cni_plugin_dirs entry: mkdir /usr/lib/cni: read-only file system

**Describe the results you expected:**
normal output of "buildah info" or "buildah images"

**Output of `rpm -q buildah` or `apt list buildah`:**

```
buildah-1.14.0-0.31.dev.git82ff48a.fc32.x86_64
```

**Output of `buildah version`:**

```
Error loading container config network config: invalid cni_plugin_dirs entry: mkdir /usr/lib/cni: read-only file system
```

**Output of `podman version` if reporting a `podman build` issue:**

```
Version:            1.8.2
RemoteAPI Version:  1
Go Version:         go1.14
OS/Arch:            linux/amd64
```

**Output of `cat /etc/*release`:**

```
NAME=Fedora
VERSION="32.20200411.n.0 (Silverblue)"
ID=fedora
VERSION_ID=32
VERSION_CODENAME=""
PLATFORM_ID="platform:f32"
PRETTY_NAME="Fedora 32.20200411.n.0 (Silverblue)"
ANSI_COLOR="0;34"
LOGO=fedora-logo-icon
CPE_NAME="cpe:/o:fedoraproject:fedora:32"
HOME_URL="https://fedoraproject.org/"
DOCUMENTATION_URL="https://docs.fedoraproject.org/en-US/fedora-silverblue/"
SUPPORT_URL="https://fedoraproject.org/wiki/Communicating_and_getting_help"
BUG_REPORT_URL="https://bugzilla.redhat.com/"
REDHAT_BUGZILLA_PRODUCT="Fedora"
REDHAT_BUGZILLA_PRODUCT_VERSION=32
REDHAT_SUPPORT_PRODUCT="Fedora"
REDHAT_SUPPORT_PRODUCT_VERSION=32
PRIVACY_POLICY_URL="https://fedoraproject.org/wiki/Legal:PrivacyPolicy"
VARIANT="Silverblue"
VARIANT_ID=silverblue
OSTREE_VERSION='32.20200411.n.0'
```

**Output of `cat /etc/containers/storage.conf`:**

```
[storage]
driver = "overlay"
runroot = "/var/run/containers/storage"
graphroot = "/var/mnt/data/containers/storage"
[storage.options]
additionalimagestores = [
]
size = ""
override_kernel_check = "true"
mountopt = "nodev,metacopy=on"
[storage.options.thinpool]
ostree_repo = ""
skip_mount_home = "false"
```

**Comment by fansari** [comment]
https://bugzilla.redhat.com/show_bug.cgi?id=1823078

**Comment by giuseppe** [comment]
it is fixed upstream with https://github.com/containers/common/commit/e8ec3e01dbea648c703967e45ee47d5bf90f24cc

**Comment by TomSweeneyRedHat** [comment]
Closing as this is fixed upstream and should also be fixed in Buildah 1.14.3 and above when containers/common v0.6.0 was vendored.

**Comment by fansari** [comment]
How can I update this on Silverblue? Today I tried again "repm-ostree upgrade" and still have buildah-1.14.0-0.31.dev.git82ff48a.fc32.x86_64 after reboot.

**Comment by rhatdan** [comment]
@dustymabe Do you have an answer?

**Comment by fansari** [comment]
This is fixed now with buildah-1.14.8-1.fc32.x86_64.
